host/testcluster.go

```diff
@@ -101,6 +101,8 @@ type (
 		EnableIndexer    bool
 		EnableReplicator bool
 	}
+
+	NoopMetricsClient struct{}
```

Comment:
Why not to use mock here too?

Reply:
I don't think we should use mocks in integration tests. Also, specifying all methods to be called inside mock is same amount of code as defining noop client and is more misleading. 
If you think using mock is still better, I can refactor this code.

Reply:
This is essentially mock too. I am ok to keep it. Please just move it to separate file.